### PR TITLE
extract styles to striveneditor.css

### PR DESCRIPTION
**Pull request by Brostafa**
https://github.com/striven-erp/striven-editor/issues/30

### CODE REVIEW

**Comment by unitehenry** [comment]
@sanjogs are you alright with these naming conventions?

**Comment by unitehenry** [comment]
I agree!

@Brostafa if you would like to implement these changes.

1. Top level ```.Striven-Editor``` css class.
2. Scoped selector ```.Striven-Editor > .body```

**Comment by Brostafa** [comment]
@unitehenry Done, let me know if you need anything else.

File: src/striveneditor.css [hunk 1]
```diff
@@ -162,15 +162,15 @@
 	background-color: #fff;
 }
 
-.metadata-section, .files-section {
+.Striven-Editor .metadata-section, .files-section {
```
Comment (sanjogs) [inline src/striveneditor.css]: .files-section is missing the scope.

Comment (Brostafa) [reply src/striveneditor.css]: @sanjogs sharp eyes! Fixed https://github.com/striven-erp/striven-editor/pull/32/commits/82119ece8353b33edb3d42d8b76263c3eaef095d

**Pull request merged**